viral/learning_stages.py

```
@@ -1,133 +1,877 @@
+            plt.tight_layout()
+            plt.savefig(
+                SERVER_PATH
+                / "viral_plots"
+                / "place_cell_heatmaps"
+                / f"{mouse_name}_{date}_rewarded_{rewarded}_bod_{bod}.png",
+                dpi=300,
+            )
+
+
+def do_stuff():
+    a = 1
```

Ruff (0.14.14)

[error] Local variable `a` is assigned to but never used

Remove assignment to unused variable `a`

(F841)